toolkit/tools/internal/userutils/userutils.go

```diff
@@ -60,13 +60,18 @@ func HashPassword(password string) (string, error) {
 	return hashedPassword, nil
 }
 
-func UserExists(username string, installChroot safechroot.ChrootInterface) (bool, error) {
+func userExistsHelper(name string, installChroot safechroot.ChrootInterface, group bool) (bool, error) {
+	typeFlag := "-u"
+	if group {
+		typeFlag = "-g"
+	}
+
 	var userExists bool
 	err := installChroot.UnsafeRun(func() error {
-		_, stderr, err := shell.Execute("id", "-u", username)
+		_, stderr, err := shell.Execute("id", typeFlag, name)
 		if err != nil {
 			if !strings.Contains(stderr, "no such user") {
```

Comment:
nit: `no such user/group`

Reply:
This is the error message returned from `id`. It is always `"no such user"` even when searching for a group.

Reply:
yes! I misread it...